### PR TITLE
Fix up includes to nlohmann json 

### PR DESCRIPTION
**Pull request by z3ntu**
Might require more changes for the changed include path

See also https://github.com/mujx/nheko/pull/448

### CODE REVIEW

**Comment by z3ntu** [comment]
And here the build fails because the docker image doesn't get built during process and therefore doesn't contain the necessary change. The Windows and Mac (`clang-format` is missing for some reason, so it actually fails) builds work though.

File: deps/CMakeLists.txt [hunk 1]
```diff
@@ -53,7 +53,7 @@ set(JSON_HEADER_URL
 set(JSON_HEADER_HASH
     ce6b5610a051ec6795fa11c33854abebb086f0fd67c311f5921c3c07f9531b44)
 
-file(DOWNLOAD ${JSON_HEADER_URL} ${DEPS_INSTALL_DIR}/include/json.hpp
+file(DOWNLOAD ${JSON_HEADER_URL} ${DEPS_INSTALL_DIR}/include/nlohmann/json.hpp
```
Comment (mujx) [inline deps/CMakeLists.txt:56]: It would be better to use the `install` target of the this project so we can also use `find_package`.

**Review by mujx — COMMENTED** [review]
Btw there already an open PR for the same issue.

**Comment by z3ntu** [comment]
Oh, didn't see the other PR. #28 looks better so I'll close this one.

**Pull request closed without merging**